Detectors/ITSMFT/ITS/tracking/include/ITStracking/Cluster.h

```diff
@@ -29,17 +29,19 @@ namespace its
 {
 
 struct Cluster final {
+  Cluster() {} // = default;
```

Comment:
I think when you comment out the C++11 initialization values below, you should comment out the whole constructor (or make it = default).
In any case, the code checker warnings should be fixed.